src/sentry/web/frontend/base.py

```diff
@@ -357,12 +358,18 @@ def is_auth_required(self, request: HttpRequest, *args: Any, **kwargs: Any) -> b
         return self.auth_required and not (request.user.is_authenticated and request.user.is_active)
 
     def handle_auth_required(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        # NOTE: initiate_login simply clears session cache
```

Comment:
I feel like this would be more helpful as a docstring for `initiate_login` rather than just a comment here

Reply:
good point. 
will update